build-system/tasks/compile-expr.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs-extra');
+const gulp = require('gulp');
+const jison = require('jison');
+
+/**
+ * Helper function that uses jison to generate a parser for the input file.
+ * @param {string} path
+ * @param {string} jisonFilename
+ * @param {string} imports
+ * @param {string} parserName
+ * @param {string} jsFilename
+ */
+function compileExpr(path, jisonFilename, imports, parserName, jsFilename) {
+  const bnf = fs.readFileSync(path + jisonFilename, 'utf8');
+  const settings = {
+    type: 'lalr',
+    debug: false,
+    moduleType: 'js',
+    'token-stack': true,
+  };
+  const generator = new jison.Generator(bnf, settings);
+  const jsModule = generator.generate(settings);
+
+  const license = fs.readFileSync(
+      'build-system/tasks/js-license.txt', 'utf8');
+  const suppressCheckTypes = '/** @fileoverview ' +
+      '@suppress {checkTypes, suspiciousCode, uselessCode} */';
+  const jsExports = 'export const ' + parserName + ' = parser;';
+
+  const out = [
+    license,
+    suppressCheckTypes,
+    imports,
+    jsModule,
+    jsExports].join('\n\n') + '\n';
```

Comment:
Right here, we can regex for `_token_stack:` and remove it from the output.

Reply:
Done. I've ensured that in the final diffs, the only material change in the generated files is that the line with `_token_stack:` is no longer present.